tracing/README.md

```diff
@@ -0,0 +1,188 @@
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing</artifactId>
+</dependency>
+<dependency>
+    <!-- to add zipkin support -->
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-zipkin</artifactId>
+</dependency>
+```
+
+code using config:
+```java
+return ServerConfiguration.builder()
+                .config(config.get("webserver"))
+                .tracer(TracerBuilder.create(config.get("tracing"))
+                                        .buildAndRegister())
+                .build();
+```
+
+To customize configuration of zipkin, see `ZipkinTracerBuilder` javadoc. Basics:
+```yaml
+tracing:
+  # required
+  service: "service-name"
+  # default is "localhost"
+  host: "zipkin"
+```
+
+### Usage of Jersey client outside of Helidon MP
+Jersey client can be used (maybe standalone, or in Helidon SE) and traced.
+Simple add `helidon-tracing-jersey-client` as a dependency and correctly configure
+the request properties. The client expects Tracer to be configured (see zipkin examples above)
+ 
+Note that in Helidon MP, this would work automatically with no additional configuration if you 
+carry out the steps described in "Usage in Helidon MP".
+
+```xml
+<dependency>
+    <groupId>io.helidon.tracing</groupId>
+    <artifactId>helidon-tracing-jersey-client</artifactId>
+</dependency>
+```
+
+And in code:
+```java
+response = webTarget
+            .request()
+            // tracer information - not required if global tracer should be used
+            .property(ClientTracingFilter.TRACER_PROPERTY_NAME, tracer)
+            // the current tracing span context to be used as a parent for outbound request
+            // if not provided a new span with no parent would be created
+            .property(ClientTracingFilter.CURRENT_SPAN_CONTEXT_PROPERTY_NAME, spanContext)
+            .get();
+```
+
+#Modules
+
+##Module `helidon-tracing`
+Contains an abstracted Builder for tracers and an SPI
+to connect various tracer implementations.
+
+Example:
+```java
+// create a tracer for service "myService"
+Tracer tracer = TracerBuilder.create("myService")
+    // running on host "zipkin" - probably in docker or k8s
+    .collectorHost("zipkin")
+    // build the tracer and register as global tracer
+    .buildAndRegister();
+```
+
+Example using Config:
+```java
+// create a tracer from configuration
+Tracer tracer = TracerBuilder.create(config.get("tracing"))    
+    // build the tracer and register as global tracer
+    .buildAndRegister();
+```
+
+and associated configuration:
+```yaml
+tracing:
+  service: "myService"
+  host: "zipkin"
+```
+
+##Module `helidon-tracing-jersey-client`
+Integration with Jersey client to add tracing support
+ for outbound requests.
+ 
+The client is registered automatically with Jersey (e.g. no need to add any filter).
+
+There are two modes of usage:
+1. Use within a Jersey server (requires `helidon-tracing-jersey` with a registered filter, or `helidon-microprofile-tracing`)
```

Comment:
Use bullets (not numbers) for list items that aren't sequential, like in this case.